src/Particles/babylon.particle.ts

```diff
@@ -61,11 +61,15 @@
          * Creates a new instance of @see Particle
          * @param particleSystem the particle system the particle belongs to
          */
-        constructor(private particleSystem: ParticleSystem) {
+        constructor(public particleSystem: ParticleSystem) {

```

Comment:
As this is a public property now you need to comment it like this:
constructor(
/** ... */
public particleSystem: ParticleSystem) {